deel/torchlip/normalizers.py

```diff
@@ -81,7 +82,9 @@ def body_cols(w):
     def body_rows(w):
         return torch.mm(torch.mm(w, w.t()), w)
 
-    def body(w, fct):
+    def body(w, fct, iter):
+        if maxiter - iter > DEFAULT_BJORK_DETACH:
+            return (1.0 + beta) * w - beta * fct(w).detach()
```

Comment:
Just out of curiosity, why the `detach` is only applied on `fct(w)` and not on the whole formula `(1+beta)*w - beta*fct(w)`? It means that the gradient is still propagated through `(1+beta)*w` at each iteration. Is it the expected behaviour?